realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.kt

```diff
@@ -716,35 +716,41 @@ class RealmProxyClassGenerator(private val processingEnvironment: ProcessingEnvi
 
             emitStatement("proxyState.getRealm\$realm().checkIfValid()")
 
-            if (forMixed) {
-                emitStatement("OsSet osSet = proxyState.getRow\$realm().getMixedSet(%s)", fieldColKeyVariableReference(field))
-            } else if (forRealmModel) {
-                emitStatement("OsSet osSet = proxyState.getRow\$realm().getModelSet(%s)", fieldColKeyVariableReference(field))
-            } else {
-                emitStatement("OsSet osSet = proxyState.getRow\$realm().getValueSet(%s, RealmFieldType.%s)", fieldColKeyVariableReference(field), Utils.getValueSetFieldType(field).name)
+            when {
+                forMixed -> {
+                    // Add Mixed logic
+                    throw UnsupportedOperationException("Missing Mixed implementation")
```

Comment:
Is this because it is not targeted for this PR or just not done yet because it is awaiting merge of another PR? If not part of this PR then link to the issue that should address this.

Reply:
Mixed is not part of this PR, it will be addressed on a separate one.